_hackbars/vikranth.md

```diff
@@ -0,0 +1,13 @@
+---
+layout: hackbar
+title: climate change
+author: vikranth
+---
+
+open source is one of the most efficient ways in which one can contribute. i personally feel it to be a great way to contribute together with different people having different ideas.
```

Comment:
hey @vikranthdatta Can you please update the line -

```suggestion
Open source is one of the most efficient ways in which one can contribute. I personally feel it to be a great way to contribute together with different people having different ideas.
```

Reply:
ok

Reply:
check it now.